src/simsopt/objectives/utilities.py

```diff
@@ -64,28 +65,52 @@ def dJ(self):
         return all_derivs
 
 
-class QuadraticPenalty(Optimizable):
+class QuadraticPenalty(Optimizable, GSONable):
 
```

Comment:
No need to add `GSONable` as a parent class. It is already a parent class.

Reply:
done